soa-suite/README.md

```diff
@@ -0,0 +1,320 @@
+# Deploy Oracle SOA Suite on Kubernetes with the WebLogic Operator
+
+The quick and easy way to deploy this chart on Oracle Kubernetes Engine (OKE) on Oracle Cloud Infrastructure (OCI) is to go check out the related Terraform template here [fixme](fixme), or the step by step guide 
+
+The Terraform template helps deploy some of the Kubernetes dependencies, however if you are deploying on other cloud providers, you will need to provision the persistent storage (as well as the database, for a production deployment) 
+
+To deploy manually, review the instructions below: 
+
+## Disclaimer
+
+This deployment of Oracle SOA Suite makes use of the Oracle SOA Suite Helm Chart based on the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release.
```

Comment:
You could combine these two sentences and the Caveats section and just have a single link to the `fmw-kubernetes` distro.

Reply:
merged and removed multiple references to fmw-kubernetes